dev-tools/mage/gomod.go

```diff
@@ -46,6 +46,9 @@ var (
 			},
 		},
 	}
+	filesToRemove = []string{
+		filepath.Join("vendor", "github.com", "yuin", "gopher-lua", "parse", "Makefile"),
```

Comment:
Are Makefiles copied by go modules? I thought it only copied go files.

Reply:
As clarified offline, go modules copy directories containing go files, with all the files. Thanks @kvch!